gnosis/safe-react
# Persist local statuses

## What it solves
Resolves #3194

## How this PR fixes it

Local statuses are now persisted in the `sessionStorage` by means of a middleware. The Redux store is initialised by said `sessionStorage` key.

## How to test it

1. Create a transaction that immediately executes/execute a queued transaction.
2. Refresh the page and observe that the status is still pending.
